backend/engine/plugins/lib/trivy_common/generate_locks.py

```diff
@@ -40,7 +40,10 @@ def check_package_files(path: str, include_dev: bool, npm_install: bool) -> tupl
     alerts = []
 
     # Find and loop through all the package.json files in the path
-    files = glob("%s/**/package.json" % path, recursive=True)
+    all_package_files = glob(f"{path}/**/package.json", recursive=True)
+
+    # Filter out paths that contain 'node_modules'
+    files = [file for file in all_package_files if "node_modules" not in file]
```

Comment:
yea conceptually we can still run out of space. I have an update to the EBS volume to push out after this one to increase the EBS volume space which is where the artemis runs. Theoretically that could run out space too though

Reply:
Kk, makes sense. Since this isn't directly preventing the error, let's keep an eye on the logs after deploying to see if it still pops up with some frequency.

Reply:
I could technically add logic to this package.lock builder to check for ebs volume space first before trying to install. Not sure how to estimate how much memory the install itself would take though but maybe that is possible?